### ISSUE TITLE
Sketchy dataset split to 104:21(train : test) by categories

### ISSUE THREAD

**Issue by develop-productivity** [qliu24/SAKE#9]
Thank you for your work. The training set and test set are divided into 104:21 by category. The search results are mentioned in the paper, but the code does not provide the corresponding WordNet-built similarity matrix ,  in other words, the 'cid_mask.Pickle' file. Can you provide one

**Comment by Qingzhang1994** [comment]
Would you like to tell me how to visualize top-K retrieval results，thank

**Comment by qliu24** [comment]
> Thank you for your work. The training set and test set are divided into 104:21 by category. The search results are mentioned in the paper, but the code does not provide the corresponding WordNet-built similarity matrix , in other words, the 'cid_mask.Pickle' file. Can you provide one

You can download the cid_mask.pickle file for Sketchy of that split from here: https://livejohnshopkins-my.sharepoint.com/:u:/g/personal/qliu24_jh_edu/EWFROi9G24xGsjOIXTdkXpgB0z1AIrJpjBZ_0OUHGA-ppA?e=U7J501

**Comment by develop-productivity** [comment]
> 

Thank you for your predecessors. But, I still have some other questions to ask you, I have contact you by email, also please grant instruction!

**Comment by develop-productivity** [comment]
> 

Thank you for your predecessors. But, I still have some other questions to ask you, I have contact you by email, also please grant instruction!

**Comment by develop-productivity** [comment]
> > Thank you for your work. The training set and test set are divided into 104:21 by category. The search results are mentioned in the paper, but the code does not provide the corresponding WordNet-built similarity matrix , in other words, the 'cid_mask.Pickle' file. Can you provide one
> 
> You can download the cid_mask.pickle file for Sketchy of that split from here: https://livejohnshopkins-my.sharepoint.com/:u:/g/personal/qliu24_jh_edu/EWFROi9G24xGsjOIXTdkXpgB0z1AIrJpjBZ_0OUHGA-ppA?e=U7J501

Thanks for your professor. But I still have some doubts consult to you and I have contact you by email, please feel free to comment!

**Comment by Mdahao** [comment]
> > > Thank you for your work. The training set and test set are divided into 104:21 by category. The search results are mentioned in the paper, but the code does not provide the corresponding WordNet-built similarity matrix , in other words, the 'cid_mask.Pickle' file. Can you provide one
> > 
> > 
> > You can download the cid_mask.pickle file for Sketchy of that split from here: https://livejohnshopkins-my.sharepoint.com/:u:/g/personal/qliu24_jh_edu/EWFROi9G24xGsjOIXTdkXpgB0z1AIrJpjBZ_0OUHGA-ppA?e=U7J501
> 
> Thanks for your professor. But I still have some doubts consult to you and I have contact you by email, please feel free to comment!

Hi,  could you provide the  training set and test set are divided into 104:21 by category?

**Comment by Mdahao** [comment]
Can you provide a source folder like[ this](https://github.com/qliu24/SAKE/tree/master/dataset/Sketchy/zeroshot1)? It's a little difficult for me to implement those

**Comment by qliu24** [comment]
The folder can be downloaded here:  [zeroshot2.zip](https://livejohnshopkins-my.sharepoint.com/:u:/g/personal/qliu24_jh_edu/EaviCEkFxZpGu8AjZi9HWkwBy2I-ei0kqnSvGtJE5lOTRQ?e=pu6EZn)
